radis/lbl/calc.py

```diff
@@ -293,6 +298,8 @@ def calc_spectrum(
                 sf.eq_spectrum(...)  #  new calculation without reloading the database
     engine : string
         Vaex or Pandas . Default Pandas, if engine is vaex memory performance is improved
+    CustomSpectrumFactory: class
```

Comment:
See my comment [there](https://github.com/radis/radis/pull/652/files/285e6832fd3c31c7abbe16fc9c7764d5eeb32a54#diff-08e99f95955a5697a50f8f766e787f3440bb5bda43ccf32f9d045005fb3284b7R92), but I very much dislike the idea of adding such a parameter in `calc_spectrum`. It kills the intent of calc_spectrum to be as simple as possible (and it isn't as simple as it should be already). Let's discuss it.